libqtile/widget/bluetooth_extra.py

```diff
@@ -0,0 +1,634 @@
+# Copyright (c) 2023 elParaguayo
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from dbus_next.aio import MessageBus
+from dbus_next.constants import BusType
+from dbus_next.errors import DBusError, InterfaceNotFoundError
+
+from libqtile.command.base import expose_command
+from libqtile.log_utils import logger
+from libqtile.utils import create_task
+from libqtile.widget import base
+
+BLUEZ_SERVICE = "org.bluez"
+BLUEZ_DEVICE = "org.bluez.Device1"
+BLUEZ_ADAPTER = "org.bluez.Adapter1"
+BLUEZ_BATTERY = "org.bluez.Battery1"
+OBJECT_MANAGER_INTERFACE = "org.freedesktop.DBus.ObjectManager"
+PROPERTIES_INTERFACE = "org.freedesktop.DBus.Properties"
+
+
+def catch_error(msg):
```

Comment:
nit: could this be `_catch_error`?

Reply:
Why?

Reply:
To indicate to others (users, contributors) that they shouldn't import from this widget module just because they want a similar function. If they do, we should factor it out into something that can be shared by both widgets